public/js/use-onepay.js

```diff
@@ -0,0 +1,55 @@
+function showLoadingImage() {
+    let html = document.getElementById("qr");
+    html.innerHTML = "";
+    let loading = new Image(200, 200);
+    loading.src = "./images/loading.gif";
+    html.appendChild(loading);
+}
+function transactionCreate() {
+    showLoadingImage();
+    $.ajax({
+        type: "POST",
+        url: "/api/transaction",
+        async: true,
+        success: function(data) {
+            // convert json to object
+            let transaction = JSON.parse(data);
+            transaction["paymentStatusHandler"] = {
```

Comment:
you can write `transaction.paymentStatusHandler = { ` here and it looks cleaner IMHO